### ISSUE TITLE
 read from one host several sites on other servers

### ISSUE THREAD

**Issue by vitaocode** [allinurl/goaccess#1593]
Hi, first of all congratulations for the great work.
I have 3 servers each server has apache with vhosts configured.
The servers are in the cloud and each access via ssh with private key.

I have a local server, I'm using this server to connect to one of the cloud servers and parse the other_vhosts_access.log of this server, works fine with this command:

    ssh -i chave.pem user@address 'tail -f -n +1 /var/log/apache2/other_vhosts_access.log' | goaccess -o /var/www/html/**server1.html** --log-format=VCOMBINED --real-time-html --hour-spec=min --enable-panel=REFERRERS --no-global-config --exclude-ip=address --real-os --ignore-crawlers --agent-list  --ignore-panel=REQUESTS_STATIC --ignore-panel=NOT_FOUND --ignore-panel=STATUS_CODES --ignore-panel=GEO_LOCATION --ignore-panel=KEYPHRASES

this way works well only for one server, but I want to monitor the other 2 servers I have

I tried to change the listening port for example:

    ssh -i chave.pem user@address 'tail -f -n +1 /var/log/apache2/other_vhosts_access.log' | goaccess -o /var/www/html/**server1.html** --log-format=VCOMBINED --real-time-html --hour-spec=min --enable-panel=REFERRERS --no-global-config --exclude-ip=address --real-os --ignore-crawlers --agent-list  --ignore-panel=REQUESTS_STATIC --ignore-panel=NOT_FOUND --ignore-panel=STATUS_CODES --ignore-panel=GEO_LOCATION --ignore-panel=KEYPHRASES **--port=7890**
 and other instance of same local server:

    ssh -i chave.pem user@address 'tail -f -n +1 /var/log/apache2/other_vhosts_access.log' | goaccess -o /var/www/html/**server2.html** --log-format=VCOMBINED --real-time-html --hour-spec=min --enable-panel=REFERRERS --no-global-config --exclude-ip=address --real-os --ignore-crawlers --agent-list  --ignore-panel=REQUESTS_STATIC --ignore-panel=NOT_FOUND --ignore-panel=STATUS_CODES --ignore-panel=GEO_LOCATION --ignore-panel=KEYPHRASES **--port=7891**

done this above, I open the browser with the pages:
    http://IPOFLOCALSERVER/server1.html and http://IPOFLOCALSERVER/server2.html

o problema acontece que as paginas .html contam apenas failed requests como na foto abaixo:

![Capturar](https://user-images.githubusercontent.com/31347646/68311350-31433480-0090-11ea-91e2-0b8ba37e5895.PNG)

I hope it was clear.
basically i monitor N servers from a single local server.
thanks in advanced

**Comment by allinurl** [comment]
Please take a look at [**this**](https://github.com/allinurl/goaccess/issues/625#issuecomment-272905761) issue, it looks like it will address your question. i.e., Different pipes (FIFOs) `--fifo-in=/path/in.1` `--fifo-out=/path/out.1`.

**Comment by vitaocode** [comment]
Works fine after created diferent pipes...thanks a lot.

**Comment by allinurl** [comment]
Glad that solved the issue. Closing this. feel free to reopen it if needed.
